src/main/java/it/gov/pagopa/pu/workflow/wf/exportfile/activity/ScheduleExportFileExpirationActivityImpl.java

```diff
@@ -0,0 +1,19 @@
+package it.gov.pagopa.pu.workflow.wf.exportfile.activity;
+
+import io.temporal.spring.boot.ActivityImpl;
+import it.gov.pagopa.pu.workflow.wf.debtposition.expirationdp.wfexpiration.CheckDebtPositionExpirationWFImpl;
+import java.time.LocalDate;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+@ActivityImpl(taskQueues = CheckDebtPositionExpirationWFImpl.TASK_QUEUE_CHECK_DP_EXPIRATION_LOCAL_ACTIVITY)
```

Comment:
this should match with the taskQueue configured for the activity when executed on the WF:
ExportFileWFConfig#buildScheduleExportFileExpirationActivityStub
```suggestion
@ActivityImpl(taskQueues = ExportFileWFImpl.TASK_QUEUE_EXPORT_FILE_LOCAL_ACTIVITY)
```